spirv_msl.cpp

```diff
@@ -4509,14 +4873,15 @@ void CompilerMSL::declare_undefined_values()
 	bool emitted = false;
 	ir.for_each_typed_id<SPIRUndef>([&](uint32_t, SPIRUndef &undef) {
 		auto &type = this->get<SPIRType>(undef.basetype);
-		statement("constant ", variable_decl(type, to_name(undef.self), undef.self), " = {};");
+		statement("constant ", CompilerGLSL::variable_decl(type, to_name(undef.self), undef.self), " = {};");
```

Comment:
Why jump up to parent class here?

Reply:
The `variable_decl` function is overloaded with two virtual functions in `CompilerGLSL`, but `CompilerMSL` only overrides one of them. Omitting the base class prefix here will fail to compile with clang.